migrations/script.py.mako

```diff
@@ -0,0 +1,22 @@
+"""${message}
```

Comment:
How is this template used? I've never been a fan the style in generated Alembic migrations, namely:

- redundant docstring
- variables before imports